### PR TITLE
Canvas effects

### PR DESCRIPTION
**Pull request by TheShovel**
Adds a bunch of canvas effects (probably my first ever stable extension)

image

### CODE REVIEW

**Comment by TheShovel** [comment]
FINALLY

**Comment by TheShovel** [comment]
Why are you so picky about formatting? What's wrong with my initial code?

File: website/index.ejs [hunk 1]
```diff
@@ -535,6 +535,12 @@
           <h3>LMS Utilities</h3>
           <p>Adds utility blocks. Created by <a href="https://scratch.mit.edu/users/LukeManiaStudios/">LukeManiaStudios</a>.</p>
         </div>
+
+        <div class="extension">
+          <%- banner('TheShovel/CanvasEffects') %>
+          <h3>Canvas Effects</h3>
+          <p>Adds utility blocks. Created by <a href="https://scratch.mit.edu/users/LukeManiaStudios/">LukeManiaStudios</a>.</p>
```
Comment (GarboMuffin) [inline website/index.ejs:542]: TODO i forgot to update this

**Comment by GarboMuffin** [comment]
I was making a few changes:

 - Change contrast from 0-1 to 0-100 like the other effects
 - Automatically reapply settings when entering fullscreen so "is canvas clear" can be removed
 - Make blocks and effects consistent with Scratch (lowercase text and menus)

Other formatting changes were coincidental in those efforts.

**Comment by GarboMuffin** [comment]
In particular this is a candidate for possibly being listed in the editor's extension library, so I'm setting the bar a bit higher

**Pull request merged**